src/tokenize_sentence/mod.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 vtext developers
+//
+// Licensed under the Apache License, Version 2.0,
+// <http://apache.org/licenses/LICENSE-2.0>. This file may not be copied,
+// modified, or distributed except according to those terms.
+
+/*!
+# Sentence tokenization module
+
+For instance let's tokenize the following text using the Unicode segmentation
+```rust
+# let s = "Here is one. Here is another! This trailing text is one more";
+# use vtext::tokenize_sentence::*;
+let tokenizer = UnicodeSentenceTokenizerParams::default();
+let tokens: Vec<&str> = tokenizer.tokenize(s).collect();
+assert_eq!(tokens, &["Here is one. ", "Here is another! ", "This trailing text is one more"];);
```

Comment:
```suggestion
let tokenizer = UnicodeSentenceTokenizer::default();
let tokens: Vec<&str> = tokenizer.tokenize(s).collect();
assert_eq!(tokens, &["Here is one. ", "Here is another! ", "This trailing text is one more"]);
```
That should fix CI failure I think?